amr-wind/core/FieldFillPatchOps.H

```diff
@@ -397,6 +402,57 @@ public:
         }
     }
 
+    void set_inflow_sibling_fields(
+        const int lev,
+        const amrex::Real time,
+        const amrex::Array<amrex::MultiFab*, AMREX_SPACEDIM> mfabs) override
+    {
+        const auto& bctype = m_field.bc_type();
+        const auto& geom = m_mesh.Geom(lev);
+        const auto& gdata = geom.data();
+        const auto& bcfunc = bc_functor();
+
+        for (amrex::OrientationIter oit; oit != nullptr; ++oit) {
+            auto ori = oit();
+            if (bctype[ori] != BC::mass_inflow) {
+                continue;
+            }
+
+            const int idir = ori.coordDir();
+            const auto& domain_box = geom.Domain();
+            for (int fdir = 0; fdir < AMREX_SPACEDIM; ++fdir) {
+
+                // Only face-normal velocities populated here
+                if (idir != fdir) {
+                    continue;
+                }
+                const auto& dbx = ori.isLow() ? amrex::bdryLo(domain_box, idir)
+                                              : amrex::bdryHi(domain_box, idir);
+
+#ifdef AMREX_USE_OMP
+#pragma omp parallel if (amrex::Gpu::notInLaunchRegion())
+#endif
+                auto& mfab = *mfabs[fdir];
```

Comment:
const

Reply:
can't do this one, mfab gets modified in the set_inflow call